constants.go

```diff
@@ -57,4 +57,10 @@ const (
 
 	// ConnectorOIDC means connector type OIDC
 	ConnectorOIDC = "oidc"
+
+	// AWSInstanceIDLabel is label specifying instance id
+	AWSInstanceIDLabel = "aws/instance-id"
+
+	// AWSInstanceRegionLabel is label specifying instance region
+	AWSInstanceRegionLabel = "aws/region"
```

Comment:
Is this a pre-defined Teleport label? I would suggest not using slash. Too frequently we try to use it as a separator. If it's a teleport label name, can we change it to simply `region`, so we can use it in the future for other clouds.